rosidl_generator_py/test/test_interfaces.py

```diff
@@ -43,9 +43,8 @@ def test_basic_types():
     assert isinstance(msg.bool_value, bool)
     assert isinstance(msg.byte_value, bytes)
     assert 1 == len(msg.byte_value)
-    # TODO(jacobperron): char_value is an int, but shouldn't it be a string?
-    # assert isinstance(msg.char_value, str)
-    # assert 1 == len(msg.char_value)
+    # for legacy reasons, 'char' interface type maps to 'uint8'
```

Comment:
Maybe make the comment more concrete by mentioning that the `char` type is being used in a `.msg` file.

Reply:
See 5816820